SurveyApp/Extensions/UIKit/UITextField/UITextField+Placeholder.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UITextField+Placeholder.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 6/8/21.
+//
+
+import UIKit
+
+extension UITextField {
+    
+    func setPlaceholderColor(color: UIColor) {
```

Comment:
What do you think about using a property with `set` instead?
```swift
var placeholderColor: UIColor {
 set { .. }
 get { .. }
}
```

Reply:
Thank you. I'll work on it.

Reply:
@markgravity I'm not sure what should be return for `placeholderColor`. Could you please help clarify and suggest me? 🙏 😅
``` Swift
var placeholderColor: UIColor {
    get {
        // What should be return?
    }
    set(newColor) {
        attributedPlaceholder = NSAttributedString(
            string: placeholder ?? "",
            attributes: [NSAttributedString.Key.foregroundColor: newColor]
        )
    }
 }
```

Reply:
@llleyelll  you can get color by this [function](https://developer.apple.com/documentation/foundation/nsattributedstring/1408174-attribute) from NSAttributedString 👍

Reply:
Thank you so much. I already refactor all text field style to be a UITextfield subclass.